iothub/service/src/Common/Data/AmqpErrorMapper.cs

```diff
@@ -194,7 +194,9 @@ public static Error ToAmqpError(Exception exception, bool includeStackTrace)
 
             string trackingId;
             error.Info.TryGetValue(IotHubAmqpProperty.TrackingId, out trackingId);
+#pragma warning disable CS0618 // Type or member is obsolete only for external dependency.
```

Comment:
TrackingHelper file should be internal only but it is not. To avoid breaking customers, it has been marked as Obsolete so we can ignore this warning here.

Reply:
sad that these warnings cannot be moved to a global suppressions file; it would be so much more cleaner (although one can argue that having the suppression description inline is more descriptive and easier to understand)!

Reply:
yeah. Ideally this should go away if we can delete the obsolete stuff sometime.

Reply:
After a major version, probably.